libs/api/domains/work-machines/src/lib/dto/getMachineParentCategoryByTypeAndModel.input.ts

```diff
@@ -0,0 +1,13 @@
+import { Field, InputType } from '@nestjs/graphql'
+import { IsString } from 'class-validator'
+
+@InputType('GetMachineParentCategoryByTypeAndModelInput')
+export class GetMachineParentCategoryByTypeAndModelInput {
+  @Field()
+  @IsString()
+  type!: string
+
+  @Field(() => String)
```

Comment:
Nitpick, but `() => String` is unnecessary

See [NestJS docs](https://docs.nestjs.com/graphql/resolvers)